python/ray/serve/replica.py

```diff
@@ -143,9 +145,6 @@ async def initialize_replica():
             self.replica = None
             self._initialize_replica = initialize_replica
 
-            # asyncio.Event used to signal that the replica is shutting down.
-            self.shutdown_event = asyncio.Event()
```

Comment:
Why did we remove the `shutdown_event`?

Reply:
This was only used to let the previous "run_forever" coroutine exit to avoid spamming an error message.

Reply:
(not needed anymore)